src/pages/sign-up.tsx

```diff
@@ -0,0 +1,3 @@
+import SignUpView from 'views/sign-up-view'
+
+export default SignUpView
```

Comment:
💭 I wonder if it still makes sense to have a 'view' in this case since it's such a standalone page. Not necessary to address but its just a thought.

Reply:
Loosely held hot take: as routing gets more complex, I feel like it might not be a bad thing to have every page have a `view` component (even for `view` components that are only used on one page). So I'd be happy either way (whether we want to move the view component into this page file, or keep it separate in `views`).

Reply:
In this case, I opted for making a view component because of how much CSS was needed and because two SVG assets were needed. This was easier to organize in a view component folder than directly in `src/pages`.